index.html

```diff
@@ -4,37 +4,172 @@
+    <section id="third-work">
+      <div class="snap">
+        <img src="images/3rd.png" width="311px" height="220px" class="snap">
+      </div>
+      <section class="all">
+        <h5 class="title">Tonic</h5>
+        <ul class="back-end">
+        <li id="canopy">CANOPY</li>
+        <li><img src="images/Counter.svg" class="dev">Back End Dev</li>
+        <li><img src="images/Counter.svg" class="dev">2015</li>
+        </ul>
+        <p class="des">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="code">
+          <li id="htm">html</li>
+          <li id="cs">css</li>
+          <li id="js">javascript</li>
+      </ul>
+        <a href="#"><button type="button" class="box">
+            <p>See Project</p>
+        </button></a>
+      </section>
+    </section>
+  </main>
+  <main class="cards-2">
+    <section id="fourth-work">
+      <div class="snap">
+        <img src="images/fourth.png" width="311px" height="220px" class="snap">
+      </div>
+      <section class="all">
+        <h5 class="title">Multi-Post<br>Stories</h5>
+        <ul class="back-end">
+        <li id="canopy">CANOPY</li>
+        <li><img src="images/Counter.svg" class="dev">Back End Dev</li>
+        <li><img src="images/Counter.svg" class="dev">2015</li>
+        </ul>
+        <p class="des">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="code">
+          <li id="htm">html</li>
+          <li id="cs">css</li>
+          <li id="js">javascript</li>
       </ul>
+        <a href="#"><button type="button" class="box">
+            <p>See Project</p>
+        </button></a>
+      </section>
     </section>
   </main>
+  <article class="about">
+    <h2>
+      About<br>Myself
+    </h2>
+    <p>
+      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <h3 id="con">LETS CONNECT</h3>
+    <ul class="socials">
+      <li><a href="#"><img src="images/Tweeter.png"></a></li>
+      <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
+      <li><a href="#"><img src="images/m.png"></a></li>
+      <li><a href="#"><img src="images/Github.png"></a></li>
+      <li><a href="#"><img src="images/tech.png"></a></li>
+    </ul>
+    <a href="#"><button type="button" id="resume" class="box">Get my resume</button></a>
+  <div class="skills">
+    <div class="fs">
+      <h6 id="la">Languages</h6>
+    <i class="fa fa-angle-down"></i>
+    </div>
+    <div class="pics">
+      <img src="images/Frame 9.png" alt="Javascript">
+      <img src="images/htm.png" alt="HTML">
+      <img src="images/Frame 11.png" alt="css">
+    </div>
+    </ul>
+    <h6 id="fra">Frameworks</h6>
+    <hr>
+    <h6 id="sk">Skills</h6>
+    <i class="fa fa-angle-right"></i>
+    <hr>
```

Comment:
Please :pray: make sure The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` tag 
like this :point_down: 
```
<ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
      </li>
      <li class="skill-card">
        <div class="skill-title"><div class="icon square"></div> Frameworks</div>
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on Rails</li>
          <li>RSPec</li>
          <li>CapyBara</li>
          <li>Selenium</li>
        </ul>
      </li>
      <li class="skill-card">
        <div class="skill-title"><div class="icon circle"></div> Skills</div>
        <ul>
          <li>Codekit</li>
          <li>Github</li>
          <li>Codepen</li>
          <li>Gitlab</li>
          <li>Terminal</li>
        </ul>
      </li>
</ul>
```